GitOut/Features/Git/Log/GitTreeEvent.cs

```diff
@@ -0,0 +1,131 @@
+﻿using System.Collections.Generic;
+using System.Windows.Media;
+
+namespace GitOut.Features.Git.Log
+{
+    public class GitTreeEvent
+    {
+        private static readonly List<Color> colors;
+        private readonly List<GitTreeNode> nodes;
+        private int colorIndex;
+
+        static GitTreeEvent() => colors = new List<Color>
+            {
+                Color.FromArgb(255, 255, 255, 0),
+                Color.FromArgb(255, 255, 200, 100),
+                Color.FromArgb(255, 0, 255, 255),
+                Color.FromArgb(255, 100, 255, 100),
+                Color.FromArgb(255, 255, 255, 255),
+                Color.FromArgb(255, 200, 200, 200),
+                Color.FromArgb(255, 100, 100, 100),
+                Color.FromArgb(255, 50, 50, 50),
+            };
+
+        public GitTreeEvent(GitHistoryEvent historyEvent, int colorIndex)
+        {
+            this.colorIndex = colorIndex;
+            Event = historyEvent;
+            CommitIndex = -1;
+            nodes = new List<GitTreeNode>();
+        }
+
+        public int CommitIndex { get; set; }
+
+        public GitHistoryEvent Event { get; }
+        public IReadOnlyCollection<GitTreeNode> Nodes => nodes;
+
+        public int ColorIndex => colorIndex;
```

Comment:
Use auto property instead of backing field
```suggestion
        public int ColorIndex { get; private set; }
```